src/app/objective/objective.component.ts

```diff
@@ -37,12 +37,24 @@ export class ObjectiveComponent implements OnInit {
   @Output() onMoveBucket = new EventEmitter<[ImmutableObjective, ImmutableObjective, ImmutableBucket]>();
   @Output() onDelete = new EventEmitter<ImmutableObjective>();
   @Output() onChanged = new EventEmitter<[ImmutableObjective, ImmutableObjective]>();
-  
+
   constructor(public dialog: MatDialog) { }
 
   ngOnInit() {
   }
 
+  hasPeopleAvailable() {
+    if (this.objective!.assignments.length) {
+      return true;
+    }
+    for (const weeks of this.unallocatedTime!.keys()) {
```

Comment:
I think this could be done more simply as follows:

```typescript
if (Array.from(this.unallocatedTime!.values()).find(t => t > 0)) {
    return true;
}
```

In fact this whole function could probably be replaced by a single line:

```typescript
return this.objective!.assignments.find(a => a.commitment > 0) || Array.from(this.unallocatedTime!.values()).find(t => t > 0);
```

Reply:
Fixed it, thanks!